applications_tests/cfd_dem_coupling/dynamic_contact_search_files/dynamic_contact_search_generator.prm

```diff
@@ -1,6 +1,6 @@
 # Listing of Parameters
-# ---------------------
-# --------------------------------------------------
+#----------------------
```

Comment:
I am not sure I understand. What's the change in this line? Why si this registering as a change if the line itself has not been changed?

Reply:
There is a hanging space between `#` an the first `-`. This space is replaced by `-`

Reply:
This file is outdated, btw. It is missing `dimension`. However, I want to fix missing `dimension` in another PR, because this one is already too big.

Reply:
I created the script I sent you on slack to fix the missing dimension, actually.